src/script/components/MessagesList/Message/ContentMessage/asset/LinkPreviewAssetComponent.tsx

```diff
@@ -116,13 +116,14 @@ const LinkPreviewAsset: React.FC<LinkPreviewAssetProps> = ({header = false, mess
                 <p>{t('conversationTweetAuthor')}</p>
               </div>
             ) : (
-              <p
+              <a
                 className="link-preview-info-link text-foreground ellipsis"
+                href={preview.url}
```

Comment:
Please use the `prependProtocol` function on the `preview.url` from `UrlUtil` to make sure the link has the right protocol. 

```
href={prependProtocol(preview.url)}
```